spec/models/metric_spec.rb

```diff
@@ -427,7 +427,7 @@
     context "with a small environment and time_profile" do
       before(:each) do
         @vm1 = FactoryGirl.create(:vm_vmware)
-        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2))
+        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2, :cpu_total_cores => 2))
```

Comment:
@bdunne It was failing because that vm used to have 2 CPUs and I just gave it 4 (duh).  How do you feel about a :cpu2x1 trait?